designer/server/src/main/scala/pl/touk/nussknacker/ui/process/test/ScenarioTestService.scala

```diff
@@ -54,24 +53,19 @@ class ScenarioTestService(testInfoProviders: ProcessingTypeDataProvider[TestInfo
 
   private implicit val timeout: Timeout = systemRequestTimeout
 
-  def getTestingCapabilities(displayableProcess: DisplayableProcess): TestingCapabilities = {
+  def getTestingCapabilities(idWithCategory: ProcessIdWithNameAndCategory, displayableProcess: DisplayableProcess): TestingCapabilities = {
```

Comment:
I think idWithCategory is not needed, it's already in displayableProcess (it's optional only temporarily)?
If we want to remain optional, we can add it to displayable "somewhere at the beginning"?